frontend/src/api/user.ts

```diff
@@ -11,8 +11,8 @@ export const changePassword = fetcher
   .path('/api/v1/users/change_password/')
   .method('post', 'application/json')
   .create();
-
 export const getMe = fetcher.path('/api/v1/users/me/').method('get').create();
+export const logout = fetcher.path('/api/v1/users/logout/').method('post').create();
```

Comment:
This method needs to be a `GET` method, because it requires a user token, which is sent in `GET` request. Otherwise, the `POST` method needs to have a token in it (so needs to be updated on Frontend side)

Reply:
The auth token is sent for all request methods: https://github.com/bugbakery/transcribee/blob/28752bc944554b4b815186fd9cd0a1622e0ad695/frontend/src/api.ts#L13

Reply:
then it is not recognized as `token: UserToken = Depends(get_user_token)`. I think in such a case argument needs to be `session: Session = Depends(get_session),`. On the local machine I checked that with GET request solution worked, while POST request - didn't work.

Reply:
`get_session` is not related to the user auth. It get the database session, i.e. the connection to the database. `get_user_token` is the correct dependency here, which works with `POST` requests. I tested this PR, it works the way it is currently implemented. We also use `POST` requests with `get_user_token` in many other places, for example when changing the password https://github.com/bugbakery/transcribee/blob/28752bc944554b4b815186fd9cd0a1622e0ad695/backend/transcribee_backend/routers/user.py#L63

Reply:
ok, cool. A double-check was necessary.